sdk/python/the0/state.py

```
@@ -0,0 +1,182 @@
+
+
+def _validate_key(key: str) -> None:
+    """Validate that a key is safe to use as a filename."""
+    if not key:
+        raise ValueError("State key cannot be empty")
+    # Prevent directory traversal
+    if "/" in key or "\\" in key or ".." in key:
+        raise ValueError("State key cannot contain path separators or '..'")
+
+
+def get(key: str, default: Any = None) -> Any:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `default`

(ANN401)